doc/doxygen/riot.doxyfile

```diff
@@ -818,13 +818,10 @@ EXCLUDE_SYMLINKS       = NO
 # exclude all test directories for example use the pattern */test/*
 
 EXCLUDE_PATTERNS       = */board/*/tools/* \
-                         */cpu/*/include/component/* \
```

Comment:
Adding Arduino Zero should be become easier now. But perhaps you can wait after I've added my SODAQ Autonomo. You can then look at how I did the board configuration.